views/email/email_partials/email_button.hbs

```diff
@@ -0,0 +1,11 @@
+
+
+  <a href="{{ href }}" style="text-decoration: none; color: #0c0c0d; padding-top:10px;">
```

Comment:
Does this need a `rel="noreferrer noopener"`?
Although, I think this may just be a link to the Monitor site's homepage.

Reply:
`noopener` is good. Referrers here might help us understand which webmail sites are sending click-thrus from the emails.